packages/components/bucket-policy/create-bucket-policy.tsx

```diff
@@ -135,11 +135,7 @@ const CreateBucketPolicy: React.FC<CreateBucketPolicyProps> = ({
       path: '/mcgms/cluster',
     },
     {
-      name: t('Buckets'),
-      path: '/mcgms/cluster/resource/noobaa.io~v1alpha1~BucketClass',
-    },
```

Comment:
why have we removed entire breadcrumb ?

Reply:
it is part of the UX where we don't have the intermediatory list page bread crumb

https://marvelapp.com/prototype/2e221bb6/screen/86409130

Reply:
we are coming to create page from the list page... so it make sense to show that in breadcrumb as well...

Reply:
let's keep it and not remove it !